docs/package_structure.md

```diff
@@ -5,29 +5,30 @@
-    - `connector` - Code regarding connectors, including connector store, connector service, connector configurations
+  - `conduit` - Defines the main runtime that ties all Conduit layers together.
+  - `connector` - Code regarding connectors, including connector store, connector service, connector configurations
       and running instances.
-    - `foundation` - Foundation contains reusable code. Should not contain any business logic. A few honorable mentions:
-        - `assert` - Exposes common assertions for testing.
-        - `cerrors` - Exposes error creation and wrapping functionality. This is the only package for errors used in Conduit.
-        - `database` - Exposes functionality for storing values.
-        - `log` - Exposes a logger. This is the logger used throughout Conduit.
-        - `metrics` - Exposes functionality for gathering and exposing metrics.
-    - `orchestrator` - Code regarding the orchestration layer.
-    - `pipeline` - Code regarding pipelines, including pipeline store, pipeline service, running pipeline instances.
-    - `plugin` - Currently contains all logic related to plugins as well as the plugins themselves. In the future a lot of
+  - `foundation` - Foundation contains reusable code. Should not contain any business logic. A few honorable mentions:
+    - `assert` - Exposes common assertions for testing.
+    - `cerrors` - Exposes error creation and wrapping functionality. This is the only package for errors used in Conduit.
+    - `database` - Exposes functionality for storing values.
+    - `log` - Exposes a logger. This is the logger used throughout Conduit.
+    - `metrics` - Exposes functionality for gathering and exposing metrics.
+  - `orchestrator` - Code regarding the orchestration layer.
+  - `pipeline` - Code regarding pipelines, including pipeline store, pipeline service, running pipeline instances.
+  - `plugin` - Currently contains all logic related to plugins as well as the plugins themselves. In the future a lot of
       this code will be extracted into separate repositories, what will be left is a plugin service that manages built-in
       and external plugins.
-    - `processor` - Provides the types for processing a `Record`. A common abbreviation for transforms is `txf`.
-        - `transform/txfbuiltin` - Contains built-in transforms.
-        - `transform/txfjs` - Provides the functionality for implementing a transform in JavaScript.
-    - `record` - Everything regarding a `Record`, that is the central entity that is pushed through a Conduit pipeline.
+  - `processor` - Provides the types for processing a `Record`. A common abbreviation for transforms is `txf`.
+    - `transform/txfbuiltin` - Contains built-in transforms.
+    - `transform/txfjs` - Provides the functionality for implementing a transform in JavaScript.
+  - `record` - Everything regarding a `Record`, that is the central entity that is pushed through a Conduit pipeline.
```

Comment:
Some info here is outdated... E.g. there's not more database package (moved to commons), also this transform stuff is outdated too. I'd say it's OK if we do it another PR.

Reply:
@hariso I can address it now

Reply:
@hariso wdyt? https://github.com/ConduitIO/conduit/pull/1732/commits/41dd15a4f65aaed13b641e500f9952a228ef2b37

Reply:
Great work!